go/cmd/dolt/cli/help.go

```diff
@@ -66,6 +68,63 @@ func PrintHelpText(commandStr, shortDesc, longDesc string, synopsis []string, pa
 	}
 }
 
+func CreateMarkdown(fs filesys.Filesys, path, commandStr, shortDesc, longDesc string, synopsis []string, parser *argparser.ArgParser) error {
+	wr, err := fs.OpenForWrite(path)
+
+	if err != nil {
+		return err
+	}
+
+	defer wr.Close()
+
+	err = iohelp.WriteIfNoErr(wr, []byte("## Command\n\n"), nil)
+	err = iohelp.WriteIfNoErr(wr, []byte(commandStr+" - "+shortDesc+"\n\n"), err)
+
+	if len(synopsis) > 0 {
+		err = iohelp.WriteIfNoErr(wr, []byte("## Synopsis\n\n"), err)
+
+		err = iohelp.WriteIfNoErr(wr, []byte("```sh\n"), err)
+		for _, synopsisLine := range synopsis {
+			err = iohelp.WriteIfNoErr(wr, []byte(commandStr+" "+synopsisLine+"\n"), err)
+		}
+		err = iohelp.WriteIfNoErr(wr, []byte("```\n\n"), err)
+	}
+
+	err = iohelp.WriteIfNoErr(wr, []byte("## Description\n\n"), err)
+	err = iohelp.WriteIfNoErr(wr, []byte(handleAngleBrackets(longDesc)+"\n\n"), err)
+
+	if len(parser.Supported) > 0 || len(parser.ArgListHelp) > 0 {
+		err = iohelp.WriteIfNoErr(wr, []byte("## Options\n\n"), nil)
```

Comment:
Not passing err into these fn calls. Intentional?

Reply:
Nope.  Bad copy pasta.  fixed